x-pack/platform/test/onechat_functional/tests/index.ts

```diff
@@ -5,13 +5,19 @@
  * 2.0.
  */
 
-import type { FtrProviderContext } from '../../functional/ftr_provider_context';
+import type { OneChatUiFtrProviderContext } from '../../onechat/services/functional';
 
-export default function ({ loadTestFile }: FtrProviderContext) {
+export default function ({ loadTestFile }: OneChatUiFtrProviderContext) {
   describe('Agent Builder', function () {
     describe('converse', function () {
       loadTestFile(require.resolve('./converse/conversation_flow.ts'));
       loadTestFile(require.resolve('./converse/conversation_history.ts'));
     });
+
+    describe('tools', function () {
+      loadTestFile(require.resolve('./tools/create_tool.ts'));
+      loadTestFile(require.resolve('./tools/landing_page.ts'));
+      loadTestFile(require.resolve('./tools/manage_tool.ts'));
+    });
   });
```

Comment:
These tools tests will start to fail if Zachary's [access PR](https://github.com/elastic/kibana/pull/237009) lands before yours. The access check relies on the user having at least 1 connector setup.  I currently do this in the `converse/` tests [here](https://github.com/elastic/kibana/blob/main/x-pack/platform/test/onechat_functional/tests/converse/conversation_flow.ts#L23-L31). If your tests start to fail because the access checks, we will have to lift the code I've shared above to be here in this file `x-pack/platform/test/onechat_functional/tests/index.ts` prior to the test suites running so that there is always a connector setup.